docs/provider/provider-setup.md

```diff
@@ -180,12 +180,13 @@ ACCOUNT_NAME=ACCOUNT_NAME
 KEYRING_BACKEND=KEYRING_BACKEND
 CHAIN_ID=CHAIN_ID
 LAVA_RPC_NODE=LAVA_RPC_NODE
+GEOLOCATION=1/2/3
```

Comment:
GEOLOCATION=GEOLOCATION
lets be consistent with the rest of the vars

Reply:
Thanks Dvir, next time you can do "suggest changes" and then it will automatically propose the change